### ISSUE TITLE
Build of 9.0.2086  fails on Linux Mint 21.1

### ISSUE THREAD

**Issue by chdiza** [vim/vim#13477]
### Steps to reproduce

Try to build from the git repo like usual.  Observe this:
```
In file included from vim9class.c:15:
vim9class.c: In function ‘ex_class’:
vim9class.c:1563:24: error: ‘e_abstract_must_be_followed_by_def_or_static’ undeclared (first use in this function); did you mean ‘e_abstract_must_be_followed_by_def’?
 1563 |                 emsg(_(e_abstract_must_be_followed_by_def_or_static));
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
vim.h:580:25: note: in definition of macro ‘_’
  580 | # define _(x) ((char *)(x))
      |                         ^
vim9class.c:1563:24: note: each undeclared identifier is reported only once for each function it appears in
 1563 |                 emsg(_(e_abstract_must_be_followed_by_def_or_static));
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
vim.h:580:25: note: in definition of macro ‘_’
  580 | # define _(x) ((char *)(x))
      |                         ^
make[1]: *** [Makefile:3510: objects/vim9class.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/opt/bld/vim/src'
make: *** [Makefile:29: first] Error 2
```

### Expected behaviour

I expect it builds.

### Version of Vim

9.0.2086

### Environment

Linux Mint 21.1

**Comment by chdiza** [comment]
Bisecting reveals that 9.0.2085 is the cause.

**Comment by chrisbra** [comment]
a little patient please 🙏 to give me time to fix it

**Comment by chdiza** [comment]
No rush :)

**Comment by tonymec** [comment]
Fixed by patch 9.0.2087
Git changeset 33c1da7ff49c3ccdce3a6d98bd620e324c5cd6ff
Mercurial changeset c29a8c1b4d0338b51405eb337170da25dcfab303

Best regards,
Tony.
